src/wallet/wallet.h

```diff
@@ -12,6 +12,7 @@
 #include <outputtype.h>
 #include <policy/feerate.h>
 #include <psbt.h>
+#include <saltedhash.h>
```

Comment:
This should be move to the next commit, no?

Reply:
Done